### PR TITLE
Added Open Service Mesh Lab

### LINKED ISSUE #2189: [Learn] Add Open Service Mesh lab
Current Behavior
The newly created Open Service Mesh lab hasn't been added to https://layer5.io/learn .

Desired Situation
Let's put up the Open Service Mesh lab [here](https://layer5.io/learn/service-mesh-labs) .

https://github.com/layer5io/layer5/blob/master/src/sections/Learn/Service-Mesh-Labs/courseData.js

https://github.com/layer5io/layer5/tree/master/src/collections/service-mesh-labs

Contributor Resources

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the master branch.

See contributing instructions
See Layer5 site designs in this Figma project. Join the Layer5 Community for access.

### PR DESCRIPTION
**Pull request by khanjasir90**
Signed-off-by: khanjasir90 <khanmohdjasir@gmail.com>

**Description**

This PR fixes #2189 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit fd26ff00735c2fabf8d6ce68ee5b4cceccc6d49c at: https://614771f0cb908dbb781d1113--layer5ng.netlify.app

File: src/sections/Learn/Service-Mesh-Labs/courseData.js [hunk 1]
```diff
@@ -62,6 +63,14 @@ export const data = [
         embed: "traefik-meshery-adapter",
         link: "/learn/service-mesh-labs/working-with-meshery-and-traefik",
         cover: traefikLogo
+      },
+      {
+        subId: "1g",
+        title: "Open Service Mesh",
+        info: "Working with Open Service Mesh",
```
Comment (warunicorn19) [inline src/sections/Learn/Service-Mesh-Labs/courseData.js:70]:
```suggestion
        info: "Learn how to run Meshery, install Open Service Mesh and deploy a sample app",
```

File: src/sections/Learn/Service-Mesh-Labs/courseData.js [hunk 2]
```diff
@@ -62,6 +63,14 @@ export const data = [
         embed: "traefik-meshery-adapter",
         link: "/learn/service-mesh-labs/working-with-meshery-and-traefik",
         cover: traefikLogo
+      },
+      {
+        subId: "1g",
+        title: "Open Service Mesh",
```
Comment (warunicorn19) [inline src/sections/Learn/Service-Mesh-Labs/courseData.js:69]:
```suggestion
        title: "Working with Meshery and Open Service Mesh",
```

**Review by warunicorn19 — CHANGES_REQUESTED** [review]

**Comment by warunicorn19** [comment]
@khanjasir90 you didn't have to close this PR😅

**Pull request closed without merging**